react/ProductHighlights.tsx

```diff
@@ -36,11 +49,34 @@ function createFilterHighlight(filter: Filter) {
   }
 }
 
-const ProductHighlights: FC<ProductHighlightsProps> = ({
+const ProductHighlightContext =
```

Comment:
You'll have to fix this lint issue

Reply:
@icazevedo

Corrections made.

Reply:
@icazevedo

I have not been able to make these corrections.

Regarding the Product Context, every time I save, it imports the Object as a type.

And the comment errors were already showing, so I modified the file earlier and passed a new structure, but last week I received a prompt to adjust everything as it was before.

You can do a test there locally, of this correction suggestion you gave me, it keeps showing Lint errors.